test/e2e/framework/pod/wait.go

```diff
@@ -362,6 +363,82 @@ func WaitForAllPodsCondition(c clientset.Interface, ns string, opts metav1.ListO
 	return pods, maybeTimeoutError(err, "waiting for at least %d pods to be %s (matched %d)", minPods, conditionDesc, matched)
 }
 
+// WaitForAllPodsRunning waits for a given `timeout` to evaluate if all the pods in given `ns` are running.
+func WaitForAllPodsRunning(c clientset.Interface, ns string, timeout time.Duration) error {
```

Comment:
should we pass in the expected number of running pods? may help uncover issues where we pass in the wrong the namespace, but the function succeeds because there were zero pods for example

Reply:
updated.